crates/api_models/src/payments.rs

```diff
@@ -453,7 +453,7 @@ pub struct PaymentsRequest {
 
     /// You can specify up to 50 keys, with key names up to 40 characters long and values up to 500 characters long. Metadata is useful for storing additional, structured information on an object.
     #[schema(value_type = Option<Object>, example = r#"{ "udf1": "some-value", "udf2": "some-value" }"#)]
-    pub metadata: Option<pii::SecretSerdeValue>,
+    pub metadata: Option<serde_json::Value>,
```

Comment:
We'd intentionally kept this a `Secret` since this was a field provided by the merchant, and to not expose possible sensitive information in application logs. If we're now removing the `Secret`, we may have to consider masking possible sensitive information to our best efforts, both in application logs and in ClickHouse events.

Reply:
adding an issue for this #5227 .
we need to add consistency for this,
either by masking/hashing/encrypting it everywhere and deciding on a proper way for this